profile/filter_test.go

```diff
@@ -15,60 +15,398 @@
 package profile
 
 import (
+	"fmt"
 	"regexp"
+	"strings"
 	"testing"
+
+	"github.com/google/pprof/internal/proftest"
 )
 
-func TestFilter(t *testing.T) {
-	// Perform several forms of filtering on the test profile.
+var simpleProfileMappings = []*Mapping{
```

Comment:
Nit: you don't have other mappings var, so simply naming it `mappings` would be sufficient.

Reply:
done